packages/components/constraints/src/horizontal/horizontal.story.js

```diff
@@ -40,20 +25,15 @@ storiesOf('Components|Constraints', module)
     },
   })
   .add('Horizontal', () => {
-    const constraint = select(
-      'constraint',
-      ['xs', 's', 'm', 'l', 'xl', 'scale'],
+    const max = select(
+      'max',
+      [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 'scale', 'auto'],
       'scale'
     );
+
     return (
-      <Horizontal constraint={constraint}>
-        <Stack>
-          {sizes.map((size) => (
-            <Row key={size.name}>
-              <InlineColorWrapper width={size.width} />
-            </Row>
-          ))}
-        </Stack>
+      <Horizontal max={max}>
+        <ColouredRow>{max.toString()}</ColouredRow>
```

Comment:
Any reason not to keep them all listed?

Reply:
All the constraint sizes? Not really, that's a good idea actually.

Previously, we were showing one constraint size (controlled by the knob) with different content sizes (width percentages), which is what was confusing, because it looked like it was all the different constraints, but it wasn't 😄

Reply:
I think we should have all the constraints listed at once, like we do with the spacings